vendor/bat-native-ledger/src/bat/ledger/internal/publisher/publisher.cc

```diff
@@ -254,7 +249,6 @@ void Publisher::SaveVisitInternal(
                           std::bind(&Publisher::onFetchFavIcon,
```

Comment:
One side effect here is that media publishers update in db but not in time to come out to publisher info so we lose favicon on first visit to media publisher. Subsequent visits show correct favicon for verifieds. Recommend follow up to trace race condition.

Reply:
Superseded by second commit